packngo.go

```diff
@@ -29,23 +29,65 @@ const (
 	headerRateReset     = "X-RateLimit-Reset"
 )
 
+type GetOptions struct {
+	Includes []string
+	Excludes []string
+}
+
 // ListOptions specifies optional global API parameters
 type ListOptions struct {
 	// for paginated result sets, page of results to retrieve
 	Page int `url:"page,omitempty"`
-
 	// for paginated result sets, the number of results to return per page
-	PerPage int `url:"per_page,omitempty"`
+	PerPage  int `url:"per_page,omitempty"`
+	Includes []string
+	Excludes []string
+}
 
-	// specify which resources you want to return as collections instead of references
-	Includes string
+func makeSureGetOptionsInclude(l *GetOptions, s string) *GetOptions {
```

Comment:
minor point, but I assume `l` is for `*ListOptions`, should `*GetOptions`here be `g`?